dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +72,34 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @Path("/{userid}/{organization}")
+    @Operation(operationId = "getUserLambdaEventsByOrganization", description = "Get all of the Lambda Events for the given user and GitHub organization.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "See OpenApi for details")
+    public List<LambdaEvent> getUserLambdaEventsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
```

Comment:
I don't know the events structure that well, but I don't think you need a new endpoint. I think you can just modify the existing `getLambdaEventsByOrganization()` endpoint, so that if the user is an admin/curator, they can get the org's events.

The downside is you won't be able to use the `@Roles` annotation. And that you may see more events than what the user sees, but I think that's OK.

Reply:
I was assuming the later as in 
1) User comes to us saying they have a problem with X workflow in Y org
2) We investigate but they haven't shared screenshots/useful information
3) We pull the logs for Y org as if we were the user in question

Reply:
I think it would be more useful to just see the org's events regardless of user -- presumably the community lead concern is typically whether workflows are getting updated or not, not whether a specific user can see them the events. But I don't feel that strongly about it, so I'll adjust my review.